Bottom: 27f0c58a00a5a686ad609bff77ae7e1ff8087a44
Top:    2a7c9bb5600112f68a6f3f8ed162e6f985ca8a4b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 00:34:45 +0000

Refresh of work-in-progress

---

diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index b7f9eea..18bc637 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -328,6 +328,7 @@ nogui_fileselect (const char *title, const char *descr,
   else
     printf ("%s : ", title);
 
+  s = fgets (buf, 1024, stdin);
   if (s == NULL || buf[0] == '\0' || buf[0] == '\r' || buf[0] == '\n')
     {
       if (default_file != NULL)
